features/step_definitions/service_steps.rb

```diff
@@ -5,7 +5,7 @@
 end
 
 Given "a service {string} of {provider}" do |name, provider|
-  provider.services.create! :name => name, :mandatory_app_key => false
+  @service = provider.services.create! :name => name, :mandatory_app_key => false
```

Comment:
As this line is changing, I'd suggest to use the syntax `name: name, mandatory_app_key: false`

Reply:
ok